src/VisualStudio/TestUtilities.Next/VisualStudioTestUtilities.Next.csproj

```diff
@@ -9,7 +9,6 @@
     <AssemblyName>Roslyn.VisualStudio.Test.Utilities.Next</AssemblyName>
     <PlatformTarget>x86</PlatformTarget>
     <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
-    <CopyNuGetImplementations>false</CopyNuGetImplementations>
```

Comment:
sure, just curiosity. what will be exactly copied? I looked other projects and some had it and some not. and test code didn't have it so I removed it.